pkg/sql/plan/build_show.go

```diff
@@ -825,6 +825,31 @@ func buildShowFunctionStatus(stmt *tree.ShowFunctionStatus, ctx CompilerContext)
 	return returnByRewriteSQL(ctx, sql, ddlType)
 }
 
+func buildShowProcedureStatus(stmt *tree.ShowProcedureStatus, ctx CompilerContext) (*Plan, error) {
+	if stmt.Like != nil && stmt.Where != nil {
+		return nil, moerr.NewSyntaxError(ctx.GetContext(), "like clause and where clause cannot exist at the same time")
+	}
+	ddlType := plan.DataDefinition_SHOW_TARGET
+	if stmt.Like != nil && stmt.Where != nil {
```

Comment:
it's the same as line 829-831?

Reply:
yea I didn't realize that. Just fixed it by combining show procedure and show function together with a bool to distinguish them.